docs/root/start/start.rst

```diff
@@ -148,6 +148,18 @@ by using a volume.
       volumes:
         - ./envoy.yaml:/etc/envoy/envoy.yaml
 
+By default the Docker image will run as the `envoy` user created at build time.
+
+The `uid` and `gid` of this user can be set at runtime using the `ENVOY_UID` and `ENVOY_GID`
+environment variables. This can be done, for example, on the Docker command line:
+
+  $ docker run -d --name envoy -e ENVOY_UID=777 -e ENVOY_GID=777 -p 9901:9901 -p 10000:10000 envoy:v1
+
+This can be useful if you wish to restrict or provide access to `unix` sockets inside the container, or
+for controlling access to an `envoy` socket from outside of the container.
+
+If you wish to run the container as the `root` user you can set `ENVOY_UID` to `0`.
```

Comment:
Nit. To use double backticks instead of the single one, e.g. for uid and gid, and other words that need to be rendered inside `<span class="pre"></span>` tag. For example:

![image](https://user-images.githubusercontent.com/73152/85917519-db243b00-b884-11ea-8e0d-949ff052bd68.png)

Reply:
updated